packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -72,26 +72,15 @@ const cicdContent = (
   // This is an example of how you can implement your company's logic in entity page.
   // You can for example enforce that all components of type 'service' should use GitHubActions
   <EntitySwitch>
```

Comment:
As we discussed, I think this should render the github actions if available **and** the terraform latest run if available. if neither of these are available, then render the empty state.

currently, it will do github actions if available **or** terraform latest run if available and then nothing if neither are available.